topology/probes/netns/netns.go

```diff
@@ -35,7 +35,7 @@ import (
 	"syscall"
 	"time"
 
-	"gopkg.in/fsnotify.v1"
+	"github.com/fsnotify/fsnotify"
```

Comment:
should be "gopkg.in/fsnotify/fsnotify.v1" as gopkg.in/fsnotify point to the old name